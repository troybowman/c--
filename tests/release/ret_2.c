../tree/ret_2.c