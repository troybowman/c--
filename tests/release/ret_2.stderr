error: non-void funcion getint must return a value
error: use of undeclared identifier something at line 5
error: line 5 - return statements in a void function must not return a value
error: line 7 - return value is not compatible with function's return type
error: non-void funcion at must return a value
error: line 12 - return statements in a non-void function must return a value
error: line 14 - return value is not compatible with function's return type
error: line 15 - return value is not compatible with function's return type
error: symbol x used a function but is not of function type, line 16
c-- returned a nonzero exit code
